scripts/release/xenon-survival.R

```diff
@@ -0,0 +1,109 @@
+create_survival_object <- function(){
+
+  dsSurvivalClient::ds.Surv(
+    time='survival$time',
+    event = 'survival$status',
+    objectname='surv_object')
+
+}
+
+verify_survival_class <- function(){
+
```

Comment:
Remove empty line (functions can start with code, no need for empty line, same goes for the other functions)